app/components/subcase/subcase-description-panel/subcase-description-view.js

```diff
@@ -21,6 +21,8 @@ export default class SubcaseDescriptionView extends Component {
   @tracked approved = null;
   @tracked modelsOfMeetings = [];
 
+  @tracked isClosed = false;
```

Comment:
Similar to previous comments, you can define a `get isFinalMeeting() { ... }` and use that instead of introducing the tracked property `isClosed`.